recipes/pyre/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyre" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/pyre/pyre/archive/v{{ version }}.tar.gz
+    sha256: a0b3fbbdcbecc4a3690c832d955a6e0b2755b92f3a440d40621911f439cf73cb
+    patches:
+        # Hard-coded version, since the auto-versioning requires a cloned repo
+        - pyre_version.patch
+
+build:
+  number: 0
+  skip: True  # [win or py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - zip
+  run:
+    - python
+
+test:
+  imports:
+    - pyre
+    - journal
+
+about:
+  home: http://pyre.orthologue.com
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: A Python framework for scientific applications
+
+  description: |
+     An open source application framework for building
+     scalable, user-friendly scientific applications.
+
+  dev_url: https://github.com/pyre/pyre
+  doc_url: http://pyre.orthologue.com/docs
+
+extra:
+  recipe-maintainers:
+    - aivazis
```

Comment:
@aivazis - are you willing to be a maintainer?

Reply:
yes, thanks